doc/Caliptra.md

```diff
@@ -1173,30 +1184,30 @@ For SoCs that intend to achieve FIPS 140-3 CMVP certification with Caliptra:
 
 The following table describes Caliptra's fuse map:
 
-*Table 19: Caliptra Fuse Map*
+*Table 20: Caliptra Fuse Map*
 
 | **Name**                        | **Size (bits)** | **ACL**         | **Fuse programming time**                       | **Description** |
 | ------------------------------- | --------------- | --------------- | ----------------------------------------------- | --------------- |
 | UDS SEED (obfuscated)           | 384             | ROM             | SoC manufacturing                               | DICE Unique Device Secret Seed. This seed is unique per device. The seed is scrambled using an obfuscation function. |
 | FIELD ENTROPY (obfuscated)      | 256             | ROM             | Device owner in-field programmable | Field-programmable by the owner, used to hedge against UDS disclosure in the supply chain. |
-| KEY MANIFEST PK HASH            | 384             | ROM FMC RUNTIME | SoC manufacturing                               | SHA384 hash of the Vendor ECDSA P384 and LMS Public Keys. |
+| KEY MANIFEST PK HASH            | 384             | ROM FMC RUNTIME | SoC manufacturing                               | SHA384 hash of the Vendor ECDSA P384 and LMS or MLDSA Public Key Descriptors. |
 | KEY MANIFEST PK HASH MASK       | 4               | ROM FMC RUNTIME | In-field programmable                           | One-hot encoded list of revoked Vendor ECDSA P384 Public Keys. |
-| OWNER PK HASH                   | 384             | ROM FMC RUNTIME | In-field programmable                           | SHA384 hash of the Owner ECDSA P384 and LMS Public Keys. |
+| OWNER PK HASH                   | 384             | ROM FMC RUNTIME | In-field programmable                           | SHA384 hash of the Owner ECDSA P384 and LMS or MLDSDA Public Keys. |
```

Comment:
@mhatrevi @varuns-nvidia Didnt we decide to move this into registers? Also should we add SVN rollback for SOC manifest PKs?

Reply:
@bharatpillilli Don't have this in my notes. Let's sync offline on this.

Reply:
We can deal with it later; Capture it in your offline notes and we close it later

Reply:
https://github.com/chipsalliance/caliptra-rtl/issues/540